supertokens/querier.go

```diff
@@ -186,6 +192,10 @@ func (q *Querier) SendDeleteRequest(path string, data map[string]interface{}, pa
 			req.Header.Set("rid", q.RIDToCore)
 		}
 
+		if querierInterceptor != nil {
+			req = querierInterceptor(req, nil)
```

Comment:
Why are you passing `nil` for the userContext? It needs to be passed in to this function call from the caller.. from the recipeInterface implementation..